schema/collectingEvent.yml

```diff
@@ -182,24 +182,69 @@ components:
             decimalLongitude:
               type: number
               description: geographic location's longitude, http://rs.tdwg.org/dwc/terms/decimalLongitude
-            verbatimCoordinates:
+            verbatimCoordinateSystem:
               type: string
-              description: verbatim original spatial coordinates of the Location, https://dwc.tdwg.org/terms/#dwc:verbatimCoordinates              
-            coordinateUncertaintyInMeters:
+              description: The spatial coordinate system for the verbatimLatitude and verbatimLongitude or the verbatimCoordinates of the Location. https://dwc.tdwg.org/list/#dwc_verbatimCoordinateSystem
+            verbatimSRS:
+              type: string
+              description: spatial reference system (SRS) upon which coordinates given in verbatimLatitude and verbatimLongitude, or verbatimCoordinates are based. https://dwc.tdwg.org/terms/#dwc:verbatimSRS
+            verbatimElevation:
```

Comment:
can't be a number, verbatim needs to be string

Reply:
I think you highlighted the wrong part, we're you referring to verbatimElevation and verbatimDepth?

Reply:
all verbatims